website/docs/r/container_ingress_secret_opaque.html.markdown

```diff
@@ -34,6 +34,7 @@ Review the argument references that you can specify for your resource.
 - `secret_name` - (Required, String) The name of the kubernetes secret.
 - `secret_namespace` - (Required, String) The namespace of the kubernetes secret.
 - `persistence`  - (Bool) Persist the secret data in your cluster. If the secret is later deleted from the command line or OpenShift web console, the secret is automatically re-created in your cluster.
+- `update_secret` - (Optional, Integer) This argument is used to force update from upstream secrets manager instance that stores secret. Increment the value to force an update to your Ingress secret for changes made to the upstream secrets manager secret. 
```

Comment:
do you mind adding a dash before the `fields` below this? I think I forgot to in the initial documentation so it's running into the next line

Reply:
You can fix the "fields" argument indentation

Reply:
fixed to be consistent with other definitions that have nested structs